src/rgw/rgw_s3select.cc

```diff
@@ -476,6 +575,21 @@ int RGWSelectObj_ObjStore_S3::handle_aws_cli_parameters(std::string& sql_query)
     boost::replace_all(m_s3select_query, LT, "<");
   }
   //AWS cli s3select parameters
+  if(m_s3select_query.find(input_tag+"><CSV") != std::string::npos)
+  {
```

Comment:
lots of style issues regarding https://google.github.io/styleguide/cppguide.html#Conditionals
* space between `if (`
* don't give `{` its own new line